alacritty_terminal/src/term/mod.rs

```diff
@@ -2077,10 +1918,11 @@ impl<T: EventListener> Handler for Term<T> {
 
     #[inline]
     fn text_area_size_pixels(&mut self) {
-        let width = self.cell_width * self.columns();
-        let height = self.cell_height * self.screen_lines();
-        let text = format!("\x1b[4;{};{}t", height, width);
-        self.event_proxy.send_event(Event::PtyWrite(text));
+        self.event_proxy.send_event(Event::TextAreaSizeRequest(Arc::new(move |win_size| {
```

Comment:
```suggestion
        self.event_proxy.send_event(Event::TextAreaSizeRequest(Arc::new(move |window_size| {
```

Just 3 characters more and still under the 100 character limit. So probably not worth to cut things here.